hera_mc/cm_hookup.py

```diff
@@ -209,7 +209,7 @@ def __get_column_header(self, hup0):
         return parts_col
 
     def __hookup_add_correlator_levels(self, hookup_dict, testing):
-        print("CM_HOOKUP[208]:  Adding correlator levels doesn't work with new pol hookup scheme yet.")
+        print("Warning:  correlator levels don't work with new pol hookup scheme yet (CM_HOOKUP[212]).")
```

Comment:
Sorry,  I should have been more clear. Issue a warning via the `warning` module, don't use the print command at all. You can see an example in `mc_session.py` on line 916.